modules/installation-requirements-user-infra-ibm-z-kvm.adoc

```diff
@@ -35,6 +35,11 @@ you install the cluster.
 To improve high availability of your cluster, distribute the control plane machines over different {op-system-base} instances on at least two physical machines.
 ====
 
+[IMPORTANT]
+====
+All virtual machines must reside in the same data store and in the same folder as the installer.
+====
+
 The bootstrap, control plane, and compute machines must use the {op-system-first} as the
```

Comment:
While you're in this file, would you mind making this change, please?
s/must use the/must use/